index.js

```diff
@@ -159,6 +163,10 @@ app.on('browser-window-blur', (e, win) => {
 	unregisterAllShortcuts(win);
 });
 
+app.on('window-all-closed', () => {
```

Comment:
I love double checks! Just put your comment above as a code comment, so future ourself  are aware this is a double check...